src/chrome/content/rules/xkcd.xml

```diff
@@ -88,15 +88,22 @@
 
         <exclusion pattern="^http://(www\.|m\.)?xkcd\.com/1663/" />
 
+		<test url="http://xkcd.com/1663/" />
+		<test url="http://m.xkcd.com/1663/" />
+		<test url="http://www.xkcd.com/1663/" />
 
-        <test url="http://xkcd.com/1663/" />
-        <test url="http://m.xkcd.com/1663/" />
-        <test url="http://www.xkcd.com/1663/" />
+	<exclusion pattern="^http://c\.xkcd\.com/" />
 
 
 	<rule from="^http://(www\.|m\.)?xkcd\.org/"
 		to="https://$1xkcd.com/" />
 
+	<rule from="^https://c\.xkcd\.com/"
+		to="http://c.xkcd.com/"
+		downgrade="1" />
+
+		<test url="https://c.xkcd.com/" />
+
```

Comment:
@Hainish (a) Right. The site fails for clients that don't already have the intermediate cached. (It's a popular CA, so almost all clients probably already do.

(b) I don't think any current clients block the insecure TLS setup. It's functional, just not safe.

Downgrading reduces security for clients that have the intermediate and aren't being actively MITMed in a certain way, but... c'est la vie. Have to be functional for everyone.